tidy3d/plugins/dispersion/fit_fast.py

```diff
@@ -0,0 +1,834 @@
+
+    min_num_poles: PositiveInt = Field(
+        1, title="Min num poles", description="Minimum number of poles used for fitting."
+    )
+    loss_bounds: Tuple[float, float] = Field(
+        (0, np.inf),
+        title="Loss bounds",
+        description="Bounds (lower, upper) on Im[eps]. Default corresponds to only passivity. "
+        "A lower bound of 0 or greater ensures passivity. To fit a gain medium without "
+        "additional constraints, use loss_bounds=(-np.inf, np.inf). "
+        "Increasing the lower bound could help with simulation stability. "
+        "A finite upper bound may be helpful when fitting lossless materials. "
+        "In this case, consider also increasing the weight for fitting the imaginary part.",
+    )
+    weights: Tuple[NonNegativeFloat, NonNegativeFloat] = Field(
+        None,
+        title="Weights",
+        description="Weights (real, imag) in objective function for fitting. The weights "
+        "are applied to the real and imaginary parts of the permittivity epsilon. The weights "
+        "must average to 1. If None, the weights are calculated according to the typical value "
+        "of the real and imaginary part, so that they are both fit well. "
+        "Changing this can be helpful if fitting either the real or imaginary part is "
+        "more important than the other.",
+    )
+    verbose: bool = Field(
+        False, title="Verbose", description="Log verbose output during fitter run."
+    )
+    relaxed: Optional[bool] = Field(
+        None,
+        title="Relaxed",
+        description="Whether to use relaxed fitting algorithm, which "
+        "has better pole relocation properties. If None, will try both original and relaxed "
+        "algorithms.",
+    )
+    smooth: Optional[bool] = Field(
+        None,
+        title="Smooth",
+        description="Whether to use real starting poles, which can "
+        "help when fitting smooth data. If None, will try both real and complex starting poles.",
+    )
+    logspacing: Optional[bool] = Field(
+        None,
+        title="Log spacing",
+        description="Whether to space the poles logarithmically. "
+        "If None, will try both log and linear spacing.",
+    )
+
+    # more technical parameters
+    num_iters: PositiveInt = Field(
+        DEFAULT_NUM_ITERS,
+        title="Number of iterations",
+        description="Number of iterations of the fitting algorithm. Make this smaller to "
+        "speed up fitter, or make it larger to try to improve fit.",
+    )
+    passivity_num_iters: PositiveInt = Field(
+        PASSIVITY_NUM_ITERS_DEFAULT,
+        title="Number of loss bounds enforcement iterations",
+        description="Number of loss bounds enforcement iterations of the fitting algorithm. "
+        "Make this smaller to speed up fitter. There will be a warning if this value "
+        "is too small. To fit a gain medium, use the loss_bounds parameter instead.",
```

Comment:
#can you put double backslash quotes around the variable names in these docstrings? Also for `None`, `True`, `False` instances in the strings.